packages/http/src/util/runtimeExpression.ts

```diff
@@ -0,0 +1,128 @@
+import { IHttpRequest, IHttpResponse } from '../types';
+import * as Option from 'fp-ts/lib/Option';
+import { lookup } from 'fp-ts/lib/Array';
+import { pipe } from 'fp-ts/lib/pipeable';
+import { get as _get } from 'lodash';
+import { pointerToPath } from '@stoplight/json';
+
+export function resolveRuntimeExpressions(input: string, request: IHttpRequest, response: IHttpResponse) {
+  return input.replace(/{(.+?)}/g, (_, expr) =>
```

Comment:
Can we put a comment on what is this RegEx going to look for? Every time I pass on this line I'm going to regex101.com to read what it is going to do :D

Reply:
Done 👍